docs/tutorials/Getting-started-with-image-classification-on-the-Raspberry-Pi/index.md

```diff
@@ -340,6 +340,12 @@ make
 cd ../..
 ```
 
+If you are running on a Raspberry Pi Zero with your custom installed OpenBLAS under /opt/OpenBLAS, change the above cmake commands to the following to make sure your model is running against correct OpenBLAS package:
```

Comment:
Will be helpful to add a link to /tutorials/Setting-up-Raspberry-Pi-Zero-W here.

Reply:
Added the link to the tutorial. Thanks!